compiler/main/driver.cpp

```diff
@@ -584,6 +587,17 @@ static void setFastFlag(const ArgumentState* state, const char* unused) {
   specializeCCode = true;
 }
 
+static void setFloatOptFlag(const ArgumentState* state, const char* unused) {
```

Comment:
It's a shame we don't have built-in arg processing to handle something like  "set to false", "not set", "set to true"

Maybe we should add that. I'm not really suggesting you have to add it for this patch, but maybe a comment above this stating that that's really what we want? I actually have no idea what'd be involved in adding that, but taking a look at arg.cpp, I can't imagine it's really straight forward.